src/index.js

```diff
@@ -619,7 +619,7 @@ class MiniCssExtractPlugin {
         }
       });
 
-      const { Template } = webpack;
+      const Template = webpack.Template || webpack.webpack.Template;
```

Comment:
`webpack.webpack` doesn't exist in `webpack` v4 and v5, also we use:
```
const webpack = compiler.webpack
      ? compiler.webpack
      : // eslint-disable-next-line global-require
        require('webpack');
```
and for v4 and v5 we have only `webpack.Template`